conference/src/main/java/com/kyriba/conference/management/service/ScheduleService.java

```diff
@@ -11,11 +13,11 @@
 {
   List<PresentationResponse> getSchedule();
 
-  long addPresentation(PresentationRequest presentation);
+  long addPresentation(@Valid PresentationRequest presentation);
```

Comment:
The same comment with validation like for another service.

Reply:
Ideally on implementation, because they will not work without proper annotation processing at implementation level.